client/src/API.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import { GetUrlsMetadataResponse } from './types';
+
+const API_URL = 'http://localhost:8000/';
+
+export const getUrlsMetadata = async (
+  urls: string[]
+): Promise<GetUrlsMetadataResponse> => {
+  const response = await axios.post(API_URL + 'urls', {
```

Comment:
1. if this takes long time to fetch, will it block the UI?
2. what happens if this throws an exception?